src/ipahealthcheck/core/core.py

```diff
@@ -264,8 +265,13 @@ def run_healthcheck(self):
             try:
                 registry.initialize(framework, config, options)
             except Exception as e:
-                logger.error("Unable to initialize %s: %s" % (name, e))
-                continue
+                warnings.warn("Trying deprecated initialization API: %s" % e,
+                              DeprecationWarning)
+                try:
+                    registry.initialize(framework, config)
+                except Exception as e:
+                    logger.error("Unable to initialize %s: %s" % (name, e))
+                    continue
```

Comment:
ACK

Reply:
oh, sorry missed to comment this PR. Looks good! Thanks!